core/src/main/java/edu/ucr/cs/riple/core/metadata/trackers/FieldRegionTracker.java

```diff
@@ -43,21 +46,21 @@ protected TrackerNode addNodeByLine(String[] values) {
   }
 
   @Override
-  public Set<Region> getRegions(Fix fix) {
+  public Optional<Set<Region>> getRegions(Fix fix) {
```

Comment:
Why `Optional<Set<Region>>` why not `Set<Region>`?  `Optional.empty` -> `Set.of()`

Reply:
Currently I am not using any `null` checker, therefore for this places I am writing it this way to make sure I am preventing NPEs. Will rewrite these in a more relaxed way as you suggested once I add NullAway to check this repo :)
Also I can't replace `Optional.empy()` with `Set.of()` since `Optional.empty()` indicates that the tracker does **not** know about the usage of the `Fix` and `Set.of()` means the tracker does know about the `Fix` and it is an empty set.